{"name": "Возврат ошибки при неправильном значение поля", "status": "passed", "description": "Проверка, что при неправильном значение поля, возвращается кода 404 и текст ошибки в теле ответа", "steps": [{"name": "Отправка POST-запроса для авторизации курьера", "status": "passed", "start": 1739104078314, "stop": 1739104078422}, {"name": "Проверка, что код ответа на создание заказа 404", "status": "passed", "start": 1739104078422, "stop": 1739104078422}, {"name": "Проверка, что в теле ответа содержится сообщение об ошибки", "status": "passed", "start": 1739104078422, "stop": 1739104078422}], "parameters": [{"name": "incorrect_field", "value": "'password'"}], "start": 1739104078314, "stop": 1739104078423, "uuid": "3a365f65-b83d-497f-acce-b726ef27e853", "historyId": "52b92c5846524393ed5bb6c2e575dc97", "testCaseId": "51569ab39f17883721bc826a2a78674c", "fullName": "tests.test_auth_courier.TestAuthCourier#test_courier_login_incorrect_field_returns_error", "labels": [{"name": "story", "value": "Логин курьера в системе"}, {"name": "feature", "value": "Работа с курьером"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_auth_courier"}, {"name": "subSuite", "value": "TestAuthCourier"}, {"name": "host", "value": "bnovo-nb-0056"}, {"name": "thread", "value": "25888-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_auth_courier"}]}